data/statuses.js

```diff
@@ -701,7 +701,7 @@ let BattleStatuses = {
 		effectType: 'Weather',
 		duration: 0,
 		onEffectiveness(typeMod, target, type, move) {
-			if (target && target.item === "utilityumbrella") return;
+			if (target && target.hasItem('utilityumbrella')) return;
```

Comment:
I think Anubis determined that the Umbrella does not interact with Delta Stream?

Reply:
It doesnt, not sure what that has to do with this line though?